docs/powergate/mainnet.md

```diff
@@ -46,27 +46,27 @@ mined block in the past	{"block-time": "2009-01-01T04:44:30.000Z",\
 "time": "2020-05-29T20:35:08.644Z", "duration": 359999438.64444387}
 ```
 
-When complete, you will have a fully functional Powergate (`powd`), a Lotus node, and an IPFS node wired correctly together to start using the Testnet!
+When complete, you will have a fully functional Powergate (`powd`), a Lotus node, and an IPFS node wired correctly together to start using the Mainnet!
 
 ### Bootstrap from a snapshot
 
-Syncing a new Lotus node from genesis can take a considerable time. The current `testnet` network is providing snapshots of the VM state every 6hrs, which means, in the worst case, syncing 6hs of new blocks.
+Syncing a new Lotus node from genesis can take a considerable time. The current `mainnet` network is providing snapshots of the VM state every 6hrs, which means, in the worst case, syncing 6hs of new blocks.
 Bootstraping the Lotus node from a snapshot implies complete trust in the party who generated the snapshot, so this is a pragmatic solution for getting up to speed fast. Depending on your use case you should consider the security risks of accepting snapshots from trusted parties.
 
-In order to boostrap from a snapshot in `testnet`, download the [snapshot CAR file](https://very-temporary-spacerace-chain-snapshot.s3.amazonaws.com/Spacerace_pruned_stateroots_snapshot_latest.car). This file is maintained up to date by Protocol Labs.
+In order to boostrap from a snapshot in `mainnet`, download the [snapshot CAR file](https://very-temporary-spacerace-chain-snapshot.s3.amazonaws.com/Spacerace_pruned_stateroots_snapshot_latest.car). This file is maintained up to date by Protocol Labs.
 We're going to assume this CAR file was downloaded to your local path: `/home/myuser/snapshots/Spacerace_pruned_stateroots_snapshot_latest.car`.
 
 You should edit the `docker/docker-compose.yaml` file adding two lines:
 ![image](https://user-images.githubusercontent.com/6136245/93375329-6383fd80-f82e-11ea-9850-2970f30a793c.png)
 
-Then run `make up`. If you inspect the Lotus node with `docker logs testnet_lotus_1`, you will see a message that is importing chain information, and a bunch of Badger compaction messages. After the importing is done, the Lotus node will continue to sync as usual showing the typical logs. At this point you should `make down` (you don't need to wait to full syncing), revert `docker/docker-compose.yaml` to the original content, and `make up` again.
+Then run `make up`. If you inspect the Lotus node with `docker logs mainnet_lotus_1`, you will see a message that is importing chain information, and many Badger compaction messages. After the importing is done, the Lotus node will continue to sync as usual showing the typical logs. At this point you should `make down` (you don't need to wait to full syncing), revert `docker/docker-compose.yaml` to the original content, and `make up` again.
```

Comment:
didn't test this live, flagging for a check

Reply:
Uhm, no. Needs some extra change un the Makefile in Powergate 'docker' folder.
Can do soon.

Reply:
okay, so will push this to docs for the time-being. let me know if you want a quick pr there

Reply:
Targeted [here](https://github.com/textileio/powergate/pull/681).